src/components/OrgInfoSegment.tsx

```diff
@@ -1,8 +1,14 @@
+'use server';
 import Image from 'next/image';
 import React, { type FC } from 'react';
 import type { SelectClub as Club } from '@src/server/db/models';
+import { api } from '@src/trpc/server';
 
-const OrgInfoSegment: FC<{ club: Club }> = ({ club }) => {
+const OrgInfoSegment: FC<{ club: Club }> = async ({ club }) => {
+  const officers = await api.club.getOfficers.query({ id: club.id });
```

Comment:
This will throw an error if not logged in and cause the page to not load

Reply:
I was going to change the api route to public, but I decided against that, so I added a new route to get info specifically for the directory, this has the added benefit that the info segment doesn't need to do any client side fetching any more.